native/src/routes/wohnen/containers/WohnenOfferContainer.js

```diff
@@ -148,12 +153,13 @@ class WohnenOfferContainer extends React.Component<WohnenPropsType, WohnenStateT
                        contentContainerStyle={{ flexGrow: 1 }}>
       <WohnenOffer title={title} offerHash={offerHash} navigateToOffer={navigateToOffer} offers={offers}
                           t={t} theme={theme} language={language} />
-      <SiteHelpfulBox navigateToFeedback={navigateToFeedback} theme={theme} t={t} />
+      <SiteHelpfulBox navigateToFeedback={navigateToFeedback} theme={theme} />
     </ScrollView>
   }
 }
 
 export default connect<PropsType, OwnPropsType, _, _, _, _>(mapStateToProps)(
+  // $FlowFixMe
```

Comment:
Add reason

Reply:
It is the wohnen offer, thats reason enough :stuck_out_tongue: